cmd/server-mux.go

```diff
@@ -28,6 +28,14 @@ import (
 	"time"
 )
 
+// The value choosen below is longest word choosen
+// from all the http verbs comprising of
+// "PRI", "OPTIONS", "GET", "HEAD", "POST",
+// "PUT", "DELETE", "TRACE", "CONNECT".
+const (
+	longestWord = 7
```

Comment:
can be `maxHTTPVerbLen` because `longestWord` is too generic a name for a global constant